src/middlewares/transform.middleware.ts

```diff
@@ -0,0 +1,58 @@
+import { NextFunction, Request, Response } from 'express';
+import { dayjs } from '@utils/dayjs.util';
+
+export const transform = () => {
+  return (body: Object, req: Request, res: Response) => {
+    const timestamp = dayjs().utc().format();
+
+    if (Array.isArray(body)) {
+      let page = Number(req.query.page);
+      let limit = Number(req.query.limit);
+
+      if (page < 1) {
+        page = 1;
+      }
+
+      const MAX_LIMIT = 2;
```

Comment:
Please move this out of the scope of this function. 

Reason for that is we don't want to keep declaring this constant whenever this function is executed.

Not that it would make a significant performance impact in this case